applications/ISO20022-Implementation-POC.md

```diff
@@ -0,0 +1,476 @@
+- **Costs:** 10400 CHF
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | GPLv3
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile that can be used to test all the functionality delivered with this milestone. |
+| 1. |  Scope and Protocol | We will define all relevant ISO20022 messages for the MVP and     - define a minimal set of XML attributes for each ISO20022 message|
+| 2. |  Blockchain | Create a Substrate chain.|
+| 3. |  Pallet | We will implement the pallet as described above.|
+
+
+
+### Milestone 2 — Bank Client for Blockchain Interaction and Substrate Integration Tests
+
+- **Estimated duration:** 2 months
+- **FTE:**  0.4 (16 person days)
+- **Costs:** 16640 CHF
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | GPLv3
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an **article** that descibes the functionality of the pallet, bank client and message bus.|
+| 1. |  Bank client | We will implement the bank client that interacts with the Substrate Chain as well as with other banks as specified above.|
+| 2. |  Interbank Message Bus | We will implement the interbank message bus as described above.|
+| 3. |  Integration Tests | Connect all parts using a running Substrate blockchain, a message bus and two customers with accounts at two banks, creating transactions and transaction reversals.|
+
+## Future Plans
+
+There are many ways in which this project can be extended in the future. Some of them are:
+
+### Protocol
+1. Allow for multiple currencies on the Polkadot Asset Hub
+2. Give customers direct access to their onchain funds
+3. Process a wider variety of ISO20022 transactions like card payments, currency conversions or even securities trading
+4. Make bank client PCI SSC compliant
+
+### Adoption
+1. Create partnerships with banks for pilot phase
```

Comment:
I see you're incorporated in Switzerland. Is that your target market? Do you have contacts in banks, and if yes, which ones would you target? If you're not confident sharing it here, you can also send us an email at grants AT web3 DOT foundation or schedule a call [here](https://calendar.app.google/puH98UkrfgEd66Ki9).

Reply:
Yes, swiss banks would be a natural place to start. Let's speak more in a call, just booked one.